packages/salesforcedx-apex-debugger/src/core/streamingService.ts

```diff
@@ -18,8 +18,8 @@ export class StreamingService {
   public static DEFAULT_TIMEOUT = 14400;
   private static instance: StreamingService;
   private readonly apiVersion = '41.0';
```

Comment:
@lcampos @DatGuyJonathan - Should we bump this up to the latest API (not in this PR but in a future one?)